nixos/modules/services/mail/postfix.nix

```diff
@@ -354,6 +328,18 @@ in
         ";
       };
 
+      config = mkOption {
+        type = with types; attrsOf (either bool (either str (listOf str)));
```

Comment:
I think this option is a good way to have some advantages from both sides. If I'd use attrs, what would `{ test = { foo = bar; }; }` be? The postfix config has flags: "yes" and "no", lists: comma seperated strings and usual strings. This type allows a good abstraction for it.